src/libserver/re_cache.c

```diff
@@ -553,14 +553,12 @@ rspamd_re_cache_check_lua_condition (struct rspamd_task *task,
 {
 	lua_State *L = (lua_State *)task->cfg->lua_state;
 	GError *err = NULL;
-	struct rspamd_lua_text *t;
 	gint text_pos;
 
 	if (G_LIKELY (lua_cbref == -1)) {
 		return TRUE;
 	}
 
-	t = lua_new_text (L, in, len, FALSE);
```

Comment:
This breaks this function... `t` is created and pushed to the Lua stack and, subsequently, accessed by varargs function `rspamd_lua_universal_pcall` - so it is a compiler bug that it emits warning here.

Reply:
Ok, thanks.  Added it back with an unused attribute.